### PR TITLE
Add new Categories UX

### PR DESCRIPTION
**Pull request by DMills94**
React checkbox tree "documentation" https://jakezatecky.github.io/react-checkbox-tree/

### CODE REVIEW

File: src/components/ManageDataModal.js [hunk 1]
```diff
@@ -279,7 +279,10 @@ function ResetDataModalContent({ setIsOpen }) {
                     type='button'
                     onClick={() => {
                         resetLocalStorageData();
-                        setSuccessText('Data has been deleted.');
+                        setSuccessText('Data has been deleted. Reloading page...');
+                        setTimeout(() => {
+                            window.location.reload();
+                        }, 1000);
```
Comment (DMills94) [inline src/components/ManageDataModal.js:285]: Oops i left this in - but given a page refresh is needed after the data has been synced let me know if you like this or not. Can always remove.

File: src/components/common/CheckboxTree.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    nodes = [],
+    onCheckFunc = () => null,
+} = {}) {
+    const [expanded, setExpanded] = useState(expandedState);
+    const dispatch = useDispatch();
+
+    const icons = {
+        check: <span className='icon-xs bg-accent w-3 h-3 rounded-sm leading-3'>check</span>,
+        uncheck: <span className='bg-inverted w-3 h-3 rounded-sm' />,
+        halfCheck: <span className='icon-xs bg-secondary w-3 h-3 rounded-sm leading-3'>check</span>,
```
Comment (DMills94) [inline src/components/common/CheckboxTree.js:18]: Noticing now we have very inconsistent checkboxes on the platform, haha

Requirements (LabelledCheckbox) - html `<input />` has default styles in some places
image

Task table checkboxes - google font icons (check_box, check_box_outline_blank)
image

What i've implemented here is a 3rd style, trying to copy the 1st `input` ones - but hard to match exactly.

What would you like to do?

File: src/components/ManageDataModal.js (hunk 1 above)
Comment (chaiinchomp) [reply src/components/ManageDataModal.js:285]: All good - this is probably good to have anyway

File: src/components/common/CheckboxTree.js (hunk 2 above)
Comment (chaiinchomp) [reply src/components/common/CheckboxTree.js:18]: Hmm yeah I think this is close enough. Can always tweak later to make things more consistent

**Review by chaiinchomp — APPROVED** [review]

**Pull request merged**